features/user/managing_customers/seeing_orders_statistics_on_customer_details_page.feature

```diff
@@ -0,0 +1,44 @@
+@customer_statistics
+Feature: Seeing customer's orders' statistics
+    In order to know how many orders a customer has placed, and what's their worth
+    As an Administrator
+    I want to be able to see customer's per channel orders' statistics on their details page
+
+    Background:
+        Given the store operates on a channel named "Web-US" in "USD" currency
+        And the store also operates on another channel named "Web-UK" in "GBP" currency
+        And the store has a product "Onion" priced at "$200" in "Web-US" channel
+        And this product is also priced at "£100" in "Web-UK" channel
+        And the store has customer "w.gandalf@sarumans.ok"
+        And the store also has customer "b.baggins@shire.me"
+        And this customer has placed 5 orders on the "Web-US" channel in each buying 12 "Onion" products
+        And this customer has also placed 12 orders on the "Web-UK" channel in each buying 20 "Onion" products
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Seeing no statistics if a customer has not placed any orders
+        When I view details of the customer "w.gandalf@sarumans.ok"
+        Then I should not see any orders statistics
+
+    @ui
+    Scenario: Seeing customer's all placed orders count
+        When I view details of the customer "b.baggins@shire.me"
+        Then I should see that this customer has placed 17 orders across all channels
```

Comment:
The feature says that the stats are going to be per channel. And here you give info for all channels. #inconsitencyAlert